### DISCUSSION TITLE
Is it currently possible to parameterize small (drug) molecules that have net charge != 0 with sage2?

### DISCUSSION THREAD

**Discussion by josephdepaoloboisvert** [openforcefield/discussions#14] [Q&A] [answered]
I attempted to parameterize a library of drug molecules by following the toolkit-showcase examples, however, only those that were neutrally charged were successfully parameterized with the openff-2.0.0 parameters.  The positively charged molecules all raised an error regrading the extra protonation of a nitrogen (the protonation of these sites is intended however).

**Comment by j-wags** [comment]
There should be no problem parametrizing molecules with net charge. Could you post the SMILES or SDF of the molecule that you're having trouble with? And could you post the error that comes out?

**Comment by davidlmobley** [comment, marked as answer]
If I had to guess on this one, I would naively guess that your nitrogens were protonated in a way which created broken valence, e.g. they had five bonds or some such. 

We routinely and frequenly use the toolkit and force fields for molecules with a formal charge without any problem.
